### PR TITLE
Ocrvs 5794 - configuration of signatures for users

### PR DESCRIPTION
**Pull request by Nil20**

### CODE REVIEW

**Comment by Nil20** [comment]
signatureTestSvg

Test svg for signature

**Comment by Zangetsu101** [comment]
@Nil20 Could you please rebase your branch from the release-v1.3.1 to get rid of the 2 extra commits that came up after changing the base branch?

File: infrastructure/emergency-backup-metadata.sh [hunk 1]
```diff
@@ -126,6 +126,14 @@ mkdir -p $ROOT_PATH/backups/mongo
 mkdir -p $ROOT_PATH/backups/minio
 mkdir -p $ROOT_PATH/backups/metabase
 mkdir -p $ROOT_PATH/backups/vsexport
+mkdir -p $ROOT_PATH/backups/metabase
```
Comment (rikukissa) [inline infrastructure/emergency-backup-metadata.sh]: These are probably not supposed to be a part of this PR, right?

File: src/data-seeding/certificates/source/test/UsingSignatureHandlebars.svg [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
...zkCFDH+HQIUAAAAAQACAAABsgIWAAwAIUAeCAEBAAFKAAAAcksDAgIBAXABTAAAAAwADBERBAwWKzMTMxMjAyYmJwYGBwMCwDS8PIQGDQMEDQeHAhb96gGAECYUFCYS/oIAAAEASAAAAbwCFAAHABtAGAABAQNdAAMDcksCAQAAcABMEREREAQMGCshIxEhESMRIQG8N/78OQF0AeT+HAIUAP//AEgAAAGFAhQCBgUEAAAAAQAuAAACCQIUABcAK0AoBgEEAgEAAQQAZwgHBQMDA3JLAAEBcAFMAAAAFwAXERETExEREwkMGysBFRQGIxUjNSImNTUzFRQWMxEzETI2NTUCCWVuNnFhN0tQNk5MAhSwWmiiomhZsa9NRgFC/r5GTa8AAAABAAL/+AGqAhQAGAB0QAoOAQMBDQEAAwJKS7APUFhAGgABAQRdAAQEcksAAABwSwADAwJfAAICeAJMG0uwHVBYQBYAAQEEXQAEBHJLAAMDAF8CAQAAcABMG0AaAAEBBF0ABARySwAAAHBLAAMDAl8AAgJ4AkxZWbcWJCYREAUMGSshIxEjDgIHBgYjIiYnNRYzMjY3PgI3IQGqOKMFDQ8KEDIzDBgJFBUcGw4IEQ8HAQsB5C5wdDVYTQUEMQs7UC52gD7//wAAAR8BdQLNAUcAJAAAAR8pmiZmAAmxAAK4AR+wMysAAAD/////AR8B4gLLAUcAiAAAAR8pmiZmAAmxAAK4AR+wMysAAAD//wA9AR8BYgLLAUcAJQAAAR8pmiZmAAmxAAO4AR+wMysAAAAAAwADAR8BgwLLABMAHAAjAGZLsCpQWEAiBgICAAkKBQMDCAADZQAHBwFdAAEBl0sACAgEXQAEBJsETBtAIAABAAcAAQdlBgICAAkKBQMDCAADZQAICARdAAQEmwRMWUAWAAAjIR8dHBoWFAATABMkERQhEQsOGSsTNTM1MzIWFRQHMxUjFhUUBiMjNTczMjY1NCYjIxEzMjU0JyMDOnlRTjBeUTBVRYsmXDwzOT5UYnRpbQHwHr0xNzsaHhk+PzvRHikoKiT+kF5QBQD//wA9AR8BjgLLAUcAJwAAAR8pmiZmAAmxAAK4AR+wMysAAAD//wA9AR8BLgLLAUcAKAAAAR8pmiZmAAmxAAG4AR+wMysAAAD//wAlAR8BFQLLAUcDcAAAAR8pmiZmAAmxA
```
Comment (rikukissa) [inline src/data-seeding/certificates/source/test/UsingSignatureHandlebars.svg]: Can we remove everything other than the signatures from this file?

Comment (Nil20) [reply src/data-seeding/certificates/source/test/UsingSignatureHandlebars.svg]: image

The svg actually doesn't work without this. So I guess, we might actually need this.

Comment (Nil20) [reply src/data-seeding/certificates/source/test/UsingSignatureHandlebars.svg]: Actually, it's okay to remove the style tag from the SVG. It works and it just shows a bolder font.

**Review by euanmillar — APPROVED** [review]

**Pull request merged**